grails-app/conf/application.groovy

```diff
@@ -501,6 +501,9 @@ app {
 if (!ala.baseURL) {
     ala.baseURL = "https://www.ala.org.au"
 }
+bie.ws.url = "https://www.bie.ala.org.au/ws"
```

Comment:
is the wrong address done intentionally?

Reply:
Thanks, fixed.  (I've also added the bie.ws.url to the ansible version of the config file)